WebService/WebServiceSocket.py

```diff
@@ -64,12 +63,13 @@ def sendError(conn, status=400, reason='', content=None):
         conn, address = web_socket.accept()
         print(f'Connection from {address}')
         while True:
-            data = conn.recv(1024).decode()
+            data = conn.recv(1024)
             if not data:
                 break
-            data = json.loads(data)
+            data = pickle.loads(data)

```

Comment:
I am not super sure about exposing the client to pickle due to the security drawbacks of pickle. https://www.synopsys.com/blogs/software-security/python-pickling/#:~:text=Dangers%20of%20Python%20pickling,data%20received%20over%20the%20network.